doc/conf_ref.rst

```diff
@@ -629,6 +629,21 @@ In addition to these hard coded variables, user-defined ones can also be used. T
 
     $ DNF_VAR_MY_VARIABLE=value
 
+To use such variable in your repository configuration remove the prefix. E.g.::
+
+    [myrepo]
+    baseurl=https://example.site/pub/fedora/$MY_VARIABLE/releases/$releasever
+
+Note that it is not possible to override ``arch`` and ``basearch`` values using neither variable files nor environmental variables.
```

Comment:
I think this sentence has a double negative in it which is not correct grammar, also a couple other improvements:

> Note that it is not possible to override **the** ``arch`` and ``basearch`` **variables** using **either** variable files **or** environmental variables.

Not entirely sure though, you might wanna double-check this if you can :slightly_smiling_face: